db/query_exercises.sql

```diff
@@ -1,12 +1,35 @@
--- Retrieve all info on all tenants
--- Retrieve the name, age, and gender of all tenants
--- Retrieve all info on all tenants older than 65
--- Retrieve all info on all tenants in apartment with id 20
--- Retrieve all info on all tenants in apartment with ids 20 or 15
--- Delete all tenants whose age is greater than 65
--- Create one new tenant, put them in any apartment you want
--- Find just the ids for all apartments
--- Find all info for apartments whose price is greater than $2300
--- Geriatric Birthday! Update all tenants whose age is 90 to be 91
--- Change all tenants ages to increase by 1
--- Find all tenants who live in an apartment that costs more than $2300
+Retrieve all info on all tenants
```

Comment:
The `--` at the from t of these lines marks them as a comment. If you leave each "exericse request" line as a comment, then you can run this file (thru psql) and see the results.